node/src/index.ts

```diff
@@ -704,6 +704,9 @@ export interface VectorIndex {
 export interface IndexStats {
   numIndexedRows: number | null
   numUnindexedRows: number | null
+  index_type: string | null
```

Comment:
Why are these needed in this PR?

Reply:
not related to the vec dim. But just noticed that we did not make changes with node sdk - so wanted to add them